vmc/resource_vmc_public_ip.go

```diff
@@ -23,7 +22,15 @@ func resourcePublicIp() *schema.Resource {
 		Update: resourcePublicIpUpdate,
 		Delete: resourcePublicIpDelete,
 		Importer: &schema.ResourceImporter{
-			State: schema.ImportStatePassthrough,
+			State: func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+				idParts := strings.Split(d.Id(), ",")
+				if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" || !IsValidUUID(idParts[1]) {
+					return nil, fmt.Errorf("Unexpected format of ID (%q), expected nsxt_reverse_proxy_url,public_ip_id", d.Id())
```

Comment:
does the API call error out if incorrect string for next_reverse_proxy_url is provided? Eg: abcd

Reply:
I can add a validation to check if the field is a valid URL